.github/workflows/receive-pr.yml

```diff
@@ -0,0 +1,46 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# --------------------------------------------------------------------------------------
+
+# This workflow will receive a PR and save the PR number for later use.
+
+name: Receive PR
```

Comment:
Shall we add an emoji here and for other steps as well :)
Something extra though

Reply:
Use the same emojies from other action files in steps like `checkout` etc

Reply:
Updated with: d86ff68cc484d08b5fa08f8c8dac7b113a223373